### DISCUSSION TITLE
Quick Launch toolbar replacement - toggle on/off?

### DISCUSSION THREAD

**Discussion by chip33** [valinet/ExplorerPatcher#826] [Ideas]
The quick launch replacement is great. Is there a downside to making the second click on an icon there close it rather than open a new instance? It seems that would be better.

**Comment by valinet** [comment]
I don't understand what you mean. Traditionally, the quick launch toolbar opened a new instance every time you clicked the icon. This mimics that as well, it's actually the intended behavior, it helps quickly opening multiple instances of an app without fumbling with pressing `shift`, the middle mouse wheel etc.

**Reply by chip33** [reply]
yes, I think it is because I am used to the open/minimise behaviour of things like chrome ,mpc or notepad++ pinned to the taskbar.
I have a few items in a quick launch too just so I can get to them quickly and this looks much better. 
I guess I meant it is a shame we can't pin some items to the new quick launch for multiple instances and keep others pinned but acting as before but that is just not possible.
